### PR TITLE
Tooling for generating GitBook API spec

### PR DESCRIPTION
**Pull request by benjaoming**
## Description

Tooling used to generate the following change: https://app.gitbook.com/o/pxmRWOPoaU8fUAbbcrus/s/oEAfhW9JLP3tJ6mPyxtF/~/changes/67/8-service-apis#undefined

Reviewers: By auto-generating these contents, I've saved A LOT of time. There's perhaps not so much to review here.

@rachellawson @conradsp if you are spending a lot of time hand-generating these sections in other Building Blocks, consider a similar approach?

## Related Issue

https://govstack-global.atlassian.net/jira/software/c/projects/CON/boards/61?modal=detail&selectedIssue=CON-156

### CODE REVIEW

File: api/govstack_csv_to_openapi.py [hunk 1]
```diff
@@ -49,18 +49,14 @@ def __missing__(self, key):
...
-  - name: individual
-    description: Individual operations
+  - name: service
+    description: Secured operations for individuals, data consumers and applications to record and verify consent
   - name: auditor
     description: Operations for external auditing systems to query detailed data from the system and subscribe to notifications.
   - name: notification
     description: Subscribe/unsubscribe notifications for data processors, consumers and frontend systems for individuals.
-  - name: callback
-    description: Callback API for other Building Blocks, especially relevant for asynchronous processes.
```
Comment (benjaoming) [inline api/govstack_csv_to_openapi.py:63]: Above changes to the OpenAPI header template were missed in #59 :see_no_evil:

**Comment by rachellawson** [comment]
Looks interesting! This outputs an OpenAPI compatible, yaml file, right?

**Comment by benjaoming** [comment]
Our OpenAPI spec is already part of the repo and the tooling, this change generates Markdown data for the GitBook section "8 - API Services" that is here: https://app.gitbook.com/o/pxmRWOPoaU8fUAbbcrus/s/oEAfhW9JLP3tJ6mPyxtF/~/changes/67/8-service-apis#undefined

**Comment by rachellawson** [comment]
I like the output! I’m not a python dev so won’t comment on the code.

Could be useful to others, though only if they create the source materials in the same way

**Comment by benjaoming** [comment]
It's based on reading the OpenAPI spec, so I think the implementation would work exactly the same way for other building blocks :+1:

**Pull request merged**